src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package com.internet.shop.db;
+
+import com.internet.shop.model.Product;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Product> products = new ArrayList<>();
+    private static Long productId = 0L;
+
+    public static void addProduct(Product product) {
+        product.setId(++productId);
```

Comment:
```suggestion
        product.setId(productId++);
```
It's okey to have the first index as zero